sql/npc_list.sql

```diff
@@ -34131,11 +34131,11 @@ INSERT INTO `npc_list` VALUES (17846760,'NPC[1e7]','NPC',0,0.000,0.000,0.000,0,5
 INSERT INTO `npc_list` VALUES (17846761,'_790','NPC',0,120.500,-0.577,-58.100,65537,50,50,9,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'SOA',1);
 INSERT INTO `npc_list` VALUES (17846762,'_791','NPC',0,120.641,-2.156,-58.100,1,50,50,9,0,0,0,4099,0x0200000000000000000000000000000000000000,0,'SOA',1);
 INSERT INTO `npc_list` VALUES (17846763,'_792','NPC',0,-138.405,-1.154,39.371,65537,50,50,8,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'SOA',1);
-INSERT INTO `npc_list` VALUES (17846764,'_793','NPC',0,-138.391,-2.248,39.380,1,50,50,8,0,0,2,4099,0x0200000000000000000000000000000000000000,0,'SOA',1);
+INSERT INTO `npc_list` VALUES (17846764,'_793','NPC',0,-138.391,-2.248,39.380,1,50,50,8,0,0,0,4099,0x0200000000000000000000000000000000000000,0,'SOA',1);
 INSERT INTO `npc_list` VALUES (17846765,'_794','NPC',0,-239.505,-0.585,-178.299,65537,50,50,8,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'SOA',1);
-INSERT INTO `npc_list` VALUES (17846766,'_795','NPC',0,-239.364,-2.164,-178.299,1,50,50,8,0,0,2,4099,0x0200000000000000000000000000000000000000,0,NULL,1);
+INSERT INTO `npc_list` VALUES (17846766,'_795','NPC',0,-239.364,-2.164,-178.299,1,50,50,8,0,0,0,4099,0x0200000000000000000000000000000000000000,0,'SOA',1);
```

Comment:
Should probably go through every npc you're using and verify they're all using the same properties (status 0, SOA content tag, NPC name). Animation could probably 8 for open since your setup function will close them on init, but will leave them open if SOA is disabled but the zone is still being used.

Reply:
Oh the floating NPC name is probably because the `flag` field is 1 and looks like should all be 65537.

Reply:
Wait sorry `entityFlags`, though they're both mismatched. Should be 6147 (missing FLAG_UNTARGETABLE).